src/test/user.spec.js

```diff
@@ -0,0 +1,42 @@
+import chai, { expect } from 'chai';
+import chaiHttp from 'chai-http';
+import app from '../index';
+
+chai.use(chaiHttp);
+
+describe('GET api/v1/users', () => {
+  it('should successfully get a list of all users', done => {
+    const userUrl = '/api/v1/users';
+    chai
+      .request(app)
+      .get(userUrl)
+      .end((error, response) => {
+        expect(response.status).to.equal(200);
+        done();
+      });
+  });
+
+  it('should ensure return a successful message', done => {
+    const userUrl = '/api/v1/users';
+    chai
+      .request(app)
+      .get(userUrl)
+      .end((error, response) => {
+        expect(response.body.message).to.equal(
+          'Users and corressponding profiles retrieved successfully'
+        );
+        done();
+      });
+  });
+
+  it('should get a list of all users paginated', done => {
```

Comment:
This doesn't test anything done in this task. Test the data returned. Test that it contains the expected fields and that they have the expected values.

Test what happens when a user passes a invalid data like `page= twenty`

Reply:
All right. I will do the proper tests. thank you.